atom-client/README.md

```diff
@@ -0,0 +1,9 @@
+Atom Client Module
```

Comment:
It would be better to have this in docs (may be expanded at some later point in time)